### DISCUSSION TITLE
Welcome to AR.js Discussions!

### DISCUSSION THREAD

**Discussion by kalwalt** [AR-js-org/AR.js#238] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

The Dicussions board is the right place to ask not strictly coding related questions, example:

> How can i do this with AR.js?

or share what you are working on, or if you want to talk about WebAR...

But please always behave appropriately, kindly and cordially. :slightly_smiling_face:  as stated in our [Code of Conduct](https://github.com/AR-js-org/AR.js/blob/91e73f483b0dd3a7c3a62d773722f7850b37ee1e/CODE_OF_CONDUCT.md)

**Comment by ghazal-barghouth** [comment]
hi how can i changing the  templeate of modules marker

**Reply by kalwalt** [reply]
Hi please open a new thread in q&a section. Thank you !
